tests/python/topi/python/test_topi_conv2d_int8.py

```diff
@@ -364,7 +365,7 @@ def get_ref_data():
         # ),
     ]
 
-    # TODO(tvm-team): Properly run ARM code on CI aarch64 environment
+    # TODO(tvm-team): Figure out ARM dot product availability on CI aarch64 environment
```

Comment:
cc @Mousius @u99127, I'd love to test the dot-product schedule on the aarch64 CI, do you know if it is supposed? Automatic detection would require `/proc/cpuinfo` etc as suggested by @u99127 in https://github.com/apache/tvm/pull/10773#discussion_r835338505, which I'd rather avoid.

Reply:
As far as I know, the CI environment should be good to run the dot-product schedules, I can take a look at cpuinfo detection later 😸

Reply:
yup I enabled the dot product test on CI, it seems to be working!

https://ci.tlcpack.ai/blue/rest/organizations/jenkins/pipelines/tvm/branches/PR-10839/runs/5/nodes/316/steps/542/log/?start=0
(Search `Running on target: llvm -device arm_cpu -mtriple aarch64-linux-gnu -mattr=+neon,+v8.2a,+dotprod`)